corediff.go

```diff
@@ -186,9 +194,14 @@ func addPath(root string, db hashDB, args *baseArgs) {
 			return nil
 		}
 
-		if !hasValidExt(path) {
-			logVerbose(grey(" - ", relPath, " (no code)"))
+		if args.AllValidText && !isValidUtf8(path) {
```

Comment:
duplicate call to isValidUtf8, could store result if necessary

Reply:
The other call is related to `args.Add`, and shouldn't be run twice if we scan or add the files AFAIK, if that's what you mean?

Reply:
you're right, i overlooked that!